morpheus/_lib/src/io/data_loader.cpp

```diff
@@ -56,8 +80,13 @@ std::shared_ptr<ControlMessage> DataLoader::load(std::shared_ptr<ControlMessage>
         {
             VLOG(5) << "Loading data using loader: " << loader_id
                     << " for message: " << control_message->config().dump() << std::endl;
-
-            loader->second->load(control_message, task);
+            try
+            {
+                loader->second->load(control_message, task);
+            } catch (std::exception& e)
+            {   
+                process_failures(e.what(), control_message, processes_failures_as_errors);
+            }
```

Comment:
Add an ellipsis catch (...), after std::exception, for unknown or non-standard errors.